lisa/tools/tpm2.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Dict, List, Optional, Sequence, Union
+
+from lisa.executable import Tool
+from lisa.operating_system import CBLMariner
+from lisa.util import LisaException
+
+
+class Tpm2(Tool):
+    @property
+    def command(self) -> str:
+        return "tpm2"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def _install(self) -> bool:
+        if isinstance(self.node.os, CBLMariner):
```

Comment:
Does it include Mariner 2.0 also? If not, check the versions too.

Reply:
This tool is available for Mariner 2.0